lib/puma/plugin/appsignal.rb

```diff
@@ -140,7 +140,7 @@ class Statsd
     def initialize
       # StatsD server location as configured in AppSignal agent StatsD server.
       @host = "127.0.0.1"
-      @port = 8125
+      @port = ENV.fetch("APPSIGNAL_STATSD_PORT", 8125)
```

Comment:
Perhaps we could add a note to the Puma docs saying that you _must_ configure the StatsD port via environment variables when using the Puma plugin.

Reply:
https://github.com/appsignal/appsignal-docs/pull/1332